README.md

```diff
@@ -16,6 +16,13 @@ Second, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/bl
 
 **If you don't read these it will take us far longer to help you with your problem.**
 
+## Contributing
+We'd love you to contribute to Homebrew. First, Please read our [Contribution guide](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
```

Comment:
If I was being anal, and let's face it, that's my MO, the `Please` shouldn't be capitalised.